webcomponents/tool/src/main/frontend/js/grader/submission.js

```diff
@@ -71,8 +71,26 @@ export class Submission {
       if (this.resubmitsAllowed === -1 || this.resubmitsAllowed > 0) {
         this.resubmitDate = moment(parseInt(init.properties.allow_resubmit_closeTime, 10)).valueOf();
       }
-
-
+      this.originalityConstants = {"originalityKey":0,"originalityLink":1,"originalityIcon":2,"originalityScore":3,"originalityName":4,"originalityInline":5,"originalityStatus":6,"originalityError":7};
+      this.originalityServiceName = init.properties["originalityServiceName"];
+      this.originalitySupplies = [];
+      for(var count=1; init.properties["originalityLink"+count]!=null; count++){
```

Comment:
this should be prob be a let instead of var?

and maybe a different variable name instead of count?